### PR TITLE
Adds FRAGILE flag for more easily damaged items

### PR DESCRIPTION
**Pull request by mugling**
Increases the damage to a worn item by a factor of either two or three and is intended to be used with fragile wearable devices

### CODE REVIEW

**Comment by illi-kun** [comment]
This PR cannot be merged in mainline in its current state because you're adding some functionality which is not actually used -- please apply new flag to any item. It will demonstrate the proper usage of the flag and allow us to test it.

**Comment by Coolthulhu** [comment]
If it only affects usage as armor, it should probably mention that in the description.

**Comment by mugling** [comment]
Now uses similar wording to other flags in the armor section

File: src/item.cpp [hunk 1]
```diff
@@ -1350,6 +1350,11 @@ std::string item::info(bool showtext, std::vector<iteminfo> &dump_ref) const
             dump->push_back(iteminfo("DESCRIPTION",
                 _("This piece of clothing is designed to protect you from harm and withstand a lot of abuse.")));
         }
+        if (is_armor() && has_flag("FRAGILE")) {
+            dump->push_back(iteminfo("DESCRIPTION", "--"));
+            dump->push_back(iteminfo("DESCRIPTION",
+                _("This item is fragile and is more easily damaged.")));
```
Comment (Coolthulhu) [inline src/item.cpp]: This line should also mention the fact that it only affects the usage as armor. Something like "this item is fragile and won't protect you for long" or something like that.

**Comment by Coolthulhu** [comment]
Damn, testing this took wayyy too much.

The flag should also make it easier to damage the items rather than making the damage more significant. In its current incarnation, it doesn't really matter.
Especially not on the items that protect the eyes.

**Pull request merged**